app/controllers/memberships_controller.rb

```diff
@@ -21,12 +21,24 @@ def show
     @membership = MembershipApplication.find(params[:id])
   end
 
+  def update
+    @membership = MembershipApplication.find(params[:id])
+    if @membership.update(membership_params)
+      flash[:notice] = 'Membership Application successfully updated'
```

Comment:
Why is the update method here. It is part of another PR. You need to keep track of what branch you branch off from. This may lead to future merge conflicts.